### ISSUE TITLE
Could you share the training loss to improve reproducibility?

### ISSUE THREAD

**Issue by xuanqing94** [google-research/FLAN#83]
Hi, thanks for sharing the datasets! I'm trying to train a flan model using t5 and other backbone models. However i'm not confident enough on how well I reproduced your results. Specifically I got much lower MMLU scores. Could you please share the training loss curve (or simply the loss at convergence?) Below is mine:
<img width="459" alt="image" src="https://github.com/google-research/FLAN/assets/8935605/9c5d4e16-35a6-4b8e-9010-1df3144d2e60">

I was using similar settings (batch size = 80, max_seq_len = 2300)
The final loss is around 0.6 after smoothing. What about the official values?

**Comment by StephennFernandes** [comment]
hey could you please let me know where can I find the scripts/.gin files to train FLAN on t5x based models ?

**Comment by xuanqing94** [comment]
@StephennFernandes I can't help you on that because I am using PyTorch based training framework.

**Comment by StephennFernandes** [comment]
you mean you used the huggingface model and fine tuned it on FLAN datasets ?

that works fine for me as well. 

btw did you get relatively similar results to what the official FLAN-T5 has ?

**Comment by xuanqing94** [comment]
I use checkpoints downloaded from huggingface, but I ran with my in-house distributed training code. 

I only tested and compared with FLAN-T5 it on MMLU dataset. It turns out that my results are way lower than the official FLAN-T5 checkpoints.
